source/crud/tests/unified/bulkWrite-deleteMany-let.yml

```
@@ -0,0 +1,92 @@
+description: "BulkWrite deleteMany-let"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: [ commandStartedEvent ]
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name crud-tests
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData: &initialData
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    documents:
+      - { _id: 1 }
+      - { _id: 2 }
+
+tests:
+  - description: "BulkWrite deleteMany with let option"
+    runOnRequirements:
+      - minServerVersion: "5.0"
+    operations:
+      - object: *collection0
+        name: bulkWrite
+        arguments:
+          requests:
+            - deleteMany:
+                filter: &filter
+                  $expr:
+                    $eq: [ "$_id", "$$id" ]
+          let: &let0
+            id: 1
```

For conciseness, you can just define this on the same line.

```suggestion
          let: &let0 { id: 1 }
```

Anything that can reduce the size of our YAML files without sacrificing readability (there's a trade-off if we try to cram too much on one line) is worthwhile, IMO.